addons/medical_treatment/functions/fnc_bandageLocal.sqf

```diff
@@ -7,28 +7,32 @@
  * 0: Patient <OBJECT>
  * 1: Body Part <STRING>
  * 2: Treatment <STRING>
+ * 3: Effectiveness coefficient <NUMBER>
  *
  * Return Value:
  * None
  *
  * Example:
- * [player, "Head", "FieldDressing"] call ace_medical_treatment_fnc_bandageLocal
+ * [player, "Head", "FieldDressing", 1] call ace_medical_treatment_fnc_bandageLocal
  *
  * Public: No
  */
 
-params ["_patient", "_bodyPart", "_bandage"];
-TRACE_3("bandageLocal",_patient,_bodyPart,_bandage);
+params ["_patient", "_bodyPart", "_bandage", ["_effectCoef", 1]];
```

Comment:
Personally I don't find the new variable name very clear, perhaps `_bandageRemaining`